src/Components/MissionForm.tsx

```diff
@@ -73,11 +81,9 @@ export const MissionForm: React.FC<iMissionFormProps> = ({ mission, setShowModal
             </select>
             <p>{ formErrors.status }</p>
             <label>Link to mission:</label>
-            <select 
-                name="linkToMission" 
-                defaultValue={formValues.linkToMission} 
-                onChange={handleChange} 
-                disabled={hasMissionsForLink}>
+            <select name="linkToMission" 
+                defaultValue={mission.parentID === null ? 'default' : mission.parentID} 
+                onChange={handleChange} disabled={mission.parentID !== null ? false : hasMissionsForLink}>
```

Comment:
At current stage (before infinite hierarchy), The select can be disabled, in the future the link to mission field needs to be enabled at any situation. (except if there's only one mission in the app I suppose).

```suggestion
                disabled={hasMissionsForLink}>
```